src/TimeGutter.jsx

```diff
@@ -1,39 +1,49 @@
-import React from 'react';
-import cn from 'classnames';
+import React, { Component, PropTypes } from 'react'
 import dates from './utils/dates';
-import localizer from './localizer'
+import TimeSliceGroup from './TimeSliceGroup.jsx'
 
-let TimeGutter = React.createClass({
+export default class TimeGutter extends Component {
+  static propTypes = {
+    step: PropTypes.number.isRequired,
+    slices: PropTypes.number.isRequired,
+    now: PropTypes.instanceOf(Date).isRequired,
+    min: PropTypes.instanceOf(Date).isRequired,
+    max: PropTypes.instanceOf(Date).isRequired,
+    hidelabels: PropTypes.bool,
+    timeGutterFormat: PropTypes.string
+  }
+  static defaultProps = {
+    step: 10,
+    slices: 2,
+    selectable: false,
+    hidelabels: false
```

Comment:
Perhaps since the common usage case is no labels that should be the default and the prob be `showLabels` instead of the negative.